csfieldguide/templates/interactives/short-term-memory-picture.html

```diff
@@ -0,0 +1,125 @@
+          </div>
+          <div id="item-soccerball" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-soccer-ball' %}
+          </div>
+
+          <div id="item-book" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-book' %}
+          </div>
+          <div id="item-toothbrush" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-electric-toothbrush' %}
+          </div>
+          <div id="item-teapot" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-teapot' %}
+          </div>
+          <div id="item-calculator" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-calculator' %}
+          </div>
+
+          <div id="item-forklift" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-fork-lift' %}
+          </div>
+          <div id="item-sock" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-socks' %}
+          </div>
+          <div id="item-cactus" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-cactus' %}
+          </div>
+
+          <div id="item-cup" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-cup' %}
+          </div>
+          <div id="item-envelope" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-envelope' %}
+          </div>
+          <div id="item-glasses" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-glasses' %}
+          </div>
+          <div id="item-backpack" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-school-backpack' %}
+          </div>
+
+          <div id="item-tshirt" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-t-shirt' %}
+          </div>
+          <div id="item-chair" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-chair' %}
+          </div>
+          <div id="item-sheep" class="stm-item col-3 d-none">
+            {% svg 'short-term-memory/icons8-sheep' %}
+          </div>
+        </div>
+      </div>
+
+      <div id="answer-input" class="col-12 text-center d-none">
+        <div class="row justify-content-center">
+          <p>{% trans "Enter the items you remember below! <span id='stm-instructions'>Please separate each item with a comma.</span>" %}</p>
+        </div>
+        <div class="row justify-content-center">
+          <textarea id="stm-answer-input" rows="4" placeholder="Enter answers here..." class="form-control col-4 my-2"></textarea>
+        </div>
+        <div class="row justify-content-center">
+          <button id="submit-button" class="btn btn-primary mb-2">{% trans "Submit" %}</button>
+        </div>
+      </div>
+
+      <p id="items-correct-text" class="text-center d-none">{% trans "Congratulations you remembered all of the items!" %}</p>
```

Comment:
Would be nice if this was green or something. Also needs a comma after congratulations